templates/mongodb-atlas.base-template.yaml

```
@@ -0,0 +1,329 @@
+---
+AWSTemplateFormatVersion: '2010-09-09'
+Description: MongoDB Atlas AWS CloudFormation Quickstart. (qs-1rkorhec7)
+Metadata:
+  cfn-lint: { config: { ignore_checks: [ W9002, W9003, W9006, E3001, E1010 ] } }
+  AWS::CloudFormation::Interface:
+    ParameterGroups:
+    - Label:
+        default: MongoDB Atlas ApiKey Configuration
+      Parameters:
+      - PublicKey
+      - PrivateKey 
+      - OrgId
+    - Label:
+        default: MongoDB Atlas Configuration
+      Parameters:
+      - ProjectName
+      - ClusterName
+      - ClusterRegion
+      - ClusterInstanceSize
+      - DatabaseUserRoleDatabaseName
+      - VPCCIDR
+      - PrivateSubnetCIDR
+      - AvailabilityZone
+      - VPCRegion
+    ParameterLabels:
+      PublicKey:
+        default: MongoDB Atlas API PublicKey
+      PrivateKey:
+        default: MongoDB Atlas API PrivateKey
+      OrgId:
+        default: MongoDB Atlas API OrgId
+      ProjectName:
+        default: Name of new Atlas Project
+      ClusterName:
+        default: Name of new cluster
+      ClusterRegion:
+        default: The AWS Region for Atlas Cluster
+      ClusterInstanceSize:
+        default: MongoDB Atlas Instance Size
+      DatabaseUserRoleDatabaseName:
+        default: MongoDB Atlas Database User Role Database Name
+      VPCCIDR:
+        default: VPC CIDR.
+      PrivateSubnetCIDR:
+        default: Private subnet CIDR
+      AvailabilityZone:
+         deafault: AZ fo Subnet
```

Typo and indentation issue.
```suggestion
       default: AZ for subnet
```